UI/tests/common/mocks/store_sessionUser_handlers.js

```diff
@@ -2,7 +2,7 @@ import { http, HttpResponse } from 'msw';
 
 export const sessionUserHandlers = [
 
-  http.get('/erp/api/v0/session', () => {
+  http.get('/erp/api/v0/session', ({ request }) => {
```

Comment:
`request` is unused? I got an unused-var error earlier.

Reply:
Removed.